### PR TITLE
Add Portal and expose listOrganizations()

### PR DESCRIPTION
**Pull request by willmanduffy**
* Returns a list of the organizations accessible to the provided
API key. Includes pagination functionality via the listMetadata
property.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -27,6 +27,7 @@
     "build": "tsc -p tsconfig.json",
     "lint": "tslint -p tsconfig.json -c tslint.json",
     "test": "jest",
+    "test:watch": "jest --watch",
```
Comment (willmanduffy) [inline package.json:30]: Just adding this as a convenience for myself to allow running tests on each file change.

File: src/portal/fixtures/list-organizations.json [hunk 2]
```diff
@@ -0,0 +1,90 @@
+{
```
Comment (willmanduffy) [inline src/portal/fixtures/list-organizations.json:1]: We haven't done this pattern yet! I wanted to test what the SDK does with the data it receives and confirm that it drops it in an acceptable data structure. Previous similar tests were just testing that we formed the correct request to the API.

It's super easy to generate these responses in Postman or, in this case, I just grabbed it from the Ruby SDK VCR cassettes.

Comment (maxchehab) [reply src/portal/fixtures/list-organizations.json:1]: I like it 👍

**Review by maxchehab — APPROVED** [review]
🚢

File: src/portal/fixtures/list-organizations.json (hunk 2 above)
Comment (maxchehab) [reply src/portal/fixtures/list-organizations.json:1]: I like it 👍

**Pull request merged**